src/main/java/scripting/idlescript/AIOMagic.java

```diff
@@ -43,11 +43,11 @@ public class AIOMagic extends IdleScript {
   final String[] bars =
       new String[] {"Bronze", "Iron", "Silver", "Steel", "Gold", "Mithril", "Adamantite", "Runite"};
 
-  final String[] jewelry = new String[] {"Amulet", "Ring"};
+  final String[] jewelry = new String[] {"Amulet", "Ring", "Crown"};
   JFrame scriptFrame = null;
   final int[] lootIds = {
     10, 169, 170, 171, 172, 173, 174, 384, 314, 315, 316, 317, 408, 522, 1314, 1315, 1316, 1317,
-    1318
+    1318, 1510, 1511, 1512, 1513, 1514
```

Comment:
Need to add 1509. Also there is some additional code required to make and deposit 1513 and 1514 (dragonstone crowns)